package.json

```diff
@@ -20,29 +17,29 @@
     "typescript should be gradually stepped up -- stable as of 2022_09_16"
   ],
   "dependencies": {
-    "@angular/animations": "^13.1.2",
-    "@angular/cdk": "^13.1.2",
-    "@angular/cli": "^13.1.3",
-    "@angular/common": "^13.1.2",
-    "@angular/compiler": "^13.1.2",
-    "@angular/compiler-cli": "^13.1.2",
-    "@angular/core": "^13.1.2",
-    "@angular/forms": "^13.1.2",
-    "@angular/localize": "^13.1.2",
-    "@angular/material": "^13.1.2",
-    "@angular/platform-browser": "^13.1.2",
-    "@angular/platform-browser-dynamic": "^13.1.2",
-    "@angular/router": "^13.1.2",
+    "@angular/animations": "^15.2.9",
+    "@angular/cdk": "^15.2.9",
+    "@angular/cli": "^15.2.8",
```

Comment:
I would be curious if there is a mismatch since this is at .8 and the others are at .9

Reply:
Good catch, there was a .9 version so I'll commit that change